javascript/src/main/components/ChannelMessages/MessageList.js

```diff
@@ -32,12 +32,12 @@ const messageContents = (text, users) => {
 export default ({ messages, users}) => {
```

Comment:
Would it be cleaner to set a default value for the `users` parameter instead? It should be a little less error-prone if more columns are added in the future.
```suggestion
export default ({ messages, users = [] }) => {
```

Reply:
Ohhh I like this ^